csharp/examples/FlightAspServerExample/FlightAspServerExample.csproj

```diff
@@ -0,0 +1,39 @@
+<!---
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License.
+-->
+
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
Though this is not important for an example project, it might be better to target `netcoreapp3.1` or `netstandard2.1` if possible because `Flight` projects are targeting them.

Reply:
Do we think most people creating flight services are using old versions of .NET? .NET Core 3.1 support ends in December of this year.

Reply:
No, I think most people will create services with newer versions than .NET Core 3.1.
It's just a matter of uniformity and I have no objection to leave as it is.
This example is useful for users using old versions of .NET even if `TargetFramework` is `net6.0`.

Reply:
> It's just a matter of uniformity

We use the `3.1` SDK to build the csharp components in the CI workflow:

https://github.com/apache/arrow/blob/049118f50ce46bf837de7bbb55486e74a9dc9e5d/.github/workflows/csharp.yml#L39-L46

So we'd never be able to build this example.

However, since .NET 6.0 has shipped, we should update our CI to use the 6.0 SDK, and then all of the example/executable projects could move up.

Reply:
We don't currently run examples in the CI, right? Should we add a CI that builds them?

Reply:
Yeah, that would be a good addition. Can you open a JIRA ticket for it?

Reply:
Yup! 😄  https://issues.apache.org/jira/browse/ARROW-15264